### PR TITLE
`dj-cmd` as resuable command

### PR DESCRIPTION
**Pull request by kilemensi**
This PR introduces `dj-cmd` package containing reusable command to start `Django`, `celery`, and `gunicorn` in `pantsbuild` monorepo.

It also serves as a starting point for discussions on how to create small, reusable, **and** useful packages in our monorepo.

### CODE REVIEW

File: dj-cmd/py/BUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        ":setup",
+        ":README",
+        ":LICENSE",
+        # Dependencies on code to be packaged into the distribution.
+        # ":lib",
+    ],
+    # generate_setup = False,
+    provides = python_artifact(
+        name = "dj-cmd",
+        version = "0.0.1",
```
Comment (esirK) [inline dj-cmd/py/BUILD:35]: What's the best way to do this versioning for all the packages that we will be building? And will we be pushing them to pip?

File: dj-cmd/py/dj_cmd/cmd.py [hunk 2]
```diff
@@ -1,26 +1,48 @@
...
+
     return gunicorn_main()
 
 
-def manage():
+def manage(settings_module):
     """Run administrative tasks."""
 
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "twoopstracker.settings")
+    if settings_module:
```
[the remark this answers is not in the document]
Comment (kilemensi) [reply dj-cmd/py/dj_cmd/cmd.py:43]: Not sure I get you...

**Review by KhadijaMahanga — APPROVED** [review]
LGTM

File: dj-cmd/py/dj_cmd/cmd.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (kilemensi) [reply dj-cmd/py/dj_cmd/cmd.py:43]: @esirK? Or are we cool with merge?

[the remark this answers is not in the document]
Comment (esirK) [reply dj-cmd/py/dj_cmd/cmd.py:43]: @kilemensi  I think we are cool. This is just a minor code structure preference.

**Review by esirK — APPROVED** [review]

**Pull request merged**